en/flight_modes/mission.md

```diff
@@ -0,0 +1,47 @@
+# MISSION Flight Mode
+
+The MISSION flight mode executes a [predefined mission/flight plan](../flying/missions.md) that has been uploaded to the flight controller. 
+
+> **Note** 
+>  * This mode requires GPS.
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Missions are usually created in a ground control station (e.g. [QGroundControl](https://docs.qgroundcontrol.com/en/PlanView/PlanView.html)) and uploaded prior to launch. They may also be created by a developer API, and/or uploaded in flight. Individual mission commands are handled in a way that is appropriate for each vehicle's flight characteristics (for example loiter is implemented as *hover* for copter and *circle* for fixed-wing). VTOL vehicles follow the behavior and parameters of fixed-wing when in FW mode, and of copter when in MC mode.
+
+At high level all vehicle types behave in the same way when MISSION mode is engaged:
+
+1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step.
+1. If a mission is stored and PX4 is landed:
+   * On copters PX4 will execute the [mission/flight plan](../flying/missions.md). If the mission does not have a `TAKEOFF` command then PX4 will add one, flying the vehicle to the minimum altitude before executing the rest of the flight plan.

```

Comment:
@hamishwillee the takeoff is not added to the mission. in stead the code checks if the vehicle is in a landed state before navigating to a waypoint and will perform a takeoff if so. that means it will also happen half way through